chroot/run.go

```diff
@@ -1161,10 +1162,21 @@ func setupChrootBindMounts(spec *specs.Spec, bundlePath string) (undoBinds func(
 			}
 		}
 		target := filepath.Join(spec.Root.Path, m.Destination)
-		if _, err := os.Stat(target); err != nil {
+		// Check if target is a symlink
+		stat, statErr := os.Lstat(target)
+		// If target is a symlink, follow the link and ensure the destination exists
+		if stat != nil && (stat.Mode()&os.ModeSymlink != 0) {
```

Comment:
If you change the statErr above to err, and then change this line to 

if err == nil && (stat.Mode()&os.ModeSymlink != 0) {

Everything will be happy including me.  :^)